code/modules/surgery/chestburster.dm

```diff
@@ -111,12 +111,12 @@
 	name = "Remove Larva"
 	desc = "extract the xenomorph larva"
 	accept_hand = TRUE
-	/*Similar to PINCH, but balanced around 100 = using bare hands. Haemostat is faster and better,
-	other tools are slower but don't burn the surgeon.*/
+	/*Using the hands to forcefully rip out the larva will be faster at the cost of damaging both the doctor and the patient, with the addition of organ damage.
+	Unlike before, the hemostat is now the best tool for removing removing the larva, as opposed to wirecutters and the fork.*/
 	tools = list(
-		/obj/item/tool/surgery/hemostat = 1.5,
-		/obj/item/tool/wirecutters = SURGERY_TOOL_MULT_SUBOPTIMAL,
-		/obj/item/tool/kitchen/utensil/fork = SURGERY_TOOL_MULT_SUBSTITUTE
+		/obj/item/tool/surgery/hemostat = 1.5, //Same as optimal (1) for 9s
+		/obj/item/tool/wirecutters = 1.8, //Same as suboptimal (1.2) for 9s
+		/obj/item/tool/kitchen/utensil/fork = 2.1 //Same as substitute (1.4) for 9s
 		)
 	time = 6 SECONDS
```

Comment:
Is there any reason to not just increase the timer by 1.5 times and then use the normal defines?
```suggestion
		/obj/item/tool/surgery/hemostat = SURGERY_TOOL_MULT_IDEAL, //Same as optimal (1) for 9s
		/obj/item/tool/wirecutters = SURGERY_TOOL_MULT_SUBOPTIMAL, //Same as suboptimal (1.2) for 9s
		/obj/item/tool/kitchen/utensil/fork = SURGERY_TOOL_MULT_SUBSTITUTE //Same as substitute (1.4) for 9s
		)
	time = 9 SECONDS
```

Reply:
It's a bit sloppy, but the reason I didn't use the normal defines was to keep the interaction speed we previously had when doing the surgery step without tools (IE: 9 seconds with hemostat, 6 seconds without any tools).

I also don't quite know enough about ss13 coding, so I opted for the solution which looks 🤨. I would have used the defines if I knew how to check whether or not a tool is being used, but that seemed like too much of an undertaking for my beginner self.

Reply:
That makes sense, doing something like this so we can still have it match the defines is probably good then.
```suggestion
		/obj/item/tool/surgery/hemostat = 1.5 * SURGERY_TOOL_MULT_IDEAL,
		/obj/item/tool/wirecutters = 1.5 * SURGERY_TOOL_MULT_SUBOPTIMAL,
		/obj/item/tool/kitchen/utensil/fork = 1.5 * SURGERY_TOOL_MULT_SUBSTITUTE
		)
	time = 6 SECONDS
```